run_pipeline_lightgcn.py

```diff
@@ -104,7 +100,7 @@ def train(config: Config):
         num_users,
         num_articles,
         embedding_dim=config.hidden_layer_size,
-        K=config.num_gnn_layers,
+        K=config.num_neighbors,
```

Comment:
This should be `n_hop_neighbors` not `num_neighbors`

Reply:
actually, shouldn't this be config.K?

Reply:
No, but this is really confusing, I'll change the naming inside lightgcn.py (and the parameter)

Reply:
done